student.rb

```diff
@@ -3,9 +3,9 @@
 class Student < Person
   attr_reader :classroom
 
-  def initialize(age, class_room, name = 'Unknown', parent_permission: true)
+  def initialize(age, parent_permission, name, classroom = 'Unknown')
```

Comment:
- [ ] You have an issue here with the parameter passed to the `initialize` method, Kindly fix it because it's causing a problem with creating a new student. 
[Hint] you can do it like that and it will fix all problems:
```ruby
def initialize(age, parent_permission = true, name="unknown", classroom)  
```